docs/howto/scim.md

```diff
@@ -74,7 +74,7 @@ To set up your IdP, you'll need the lakeFS Cloud SCIM provisioning endpoint and
 In the Entra ID admin dashboard, go to **Enterprise Applications** and choose the lakeFS Cloud enterprise application from the list. Then click **Provisioning** in the sidebar and then **Get Started**.
 
 1. In the provisioning settings set mode to **Automatic**
-2. In **Tenant URL** enter the URL from the lakeFS Cloud provisioning settings
+2. In **Tenant URL** enter the URL from the lakeFS Cloud provisioning settings. You will need to append `?aadOptscim062020` to the end of the URL to ensure proper integration with Entra ID.
```

Comment:
Probably a good idea to open an issue to remove this in a couple of months when this becomes the default